server.js

```diff
@@ -30,16 +34,11 @@ app.action(
     ack();
     next();
   },
-  async ({
-    action: { action_id },
-    body: {
-      user: { id: user },
-      channel: { id: channel },
-      message: { ts, blocks }
-    }
-  }) => {
-    markChoreDone(action_id, user, channel, ts, blocks);
+  async ({ action, body }) => {
+    if (!action || !body || !body.user || !body.channel || !body.message) return;
```

Comment:
Do we need to add anything for Sentry here?

Reply:
In the future, we can add the following for all functions:

```js
Sentry.configureScope(scope => {
  scope.setUser({ username: "[username]" });
  scope.setExtra("key", "value");
  // etc...
});
```